actions/stripe-mock/action.yml

```
@@ -0,0 +1,17 @@
+name: 'stripe-mock'
+description: 'Starts stripe-mock'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Start stripe-mock
+      shell: bash
+      run: |
+        BETA_ARG=""
+
+        if [ "${{github.base_ref || github.ref_name}}" == "beta" ]; then
+          BETA_ARG="-beta"
```

```suggestion
          BETA_ARG="--beta"
```
Does this need to be changed?

---

Nope, go is special :)